### ISSUE TITLE
confignetwork script does not add bonding options from nicsextraparams field from nics table

### ISSUE THREAD

**Issue by conxuro** [xcat2/xcat-core#6587]
Not sure if it is not working or if it is a feature not implemented yet (if so, would be nice to be added).

Adding BONDING_OPTS to nicextraparams field in nics table it is ignored, resulting in a bonding configured with some kind of xCAT defaults, and the impossibility to change this values with confignetwork postscript.

Example:
* Nics table nicsextraparams values:
```
$ lsdef nodo20 | grep extra
    nicextraparams.enp1s0f0=MTU=9000
    nicextraparams.enp1s0f1=MTU=9000
    nicextraparams.bond0=MTU=9000
    nicextraparams.bond0=BONDING_OPTS='lacp_rate=1'
```

* After the installation:
```
[root@nodo20 ~]# egrep "OPTS|MTU"  /etc/sysconfig/network-scripts/ifcfg-bond0
BONDING_OPTS="mode=802.3ad miimon=100"
MTU="9000"
```

The expected behaviour will be to add the custom options to be able to tune the bonding, like the script does with MTU.

---

Software used:

* Management node
```
CentOS Linux release 7.7.1908 (Core)
$ xcatconfig -v
Version 2.15 (git commit 218c6d3acc8bdbd7f72115e48cda2b1a3613d18a, built Mon Nov  4 15:17:59 EST 2019)
```
* Compute node
```
[root@nodo20 ~]# cat /etc/redhat-release 
CentOS Linux release 8.0.1905 (Core)
```

**Comment by gurevichmark** [comment]
@conxuro Have you tried `"` instead of `'` ?
Also, might be related to parsing reported by #6494

**Comment by conxuro** [comment]
I have just tried with these three options:
```
$ chdef nodo20 nicextraparams.bond0='BONDING_OPTS=mode=802.3ad miimon=100 lacp_rate=1'
$ chdef nodo20 nicextraparams.bond0='BONDING_OPTS='mode=802.3ad miimon=100 lacp_rate=1''
$ chdef nodo20 nicextraparams.bond0='BONDING_OPTS="mode=802.3ad miimon=100 lacp_rate=1"'
```
And all three options results in the same configuration, that seems are ignored and configured with xCAT defaults:
```
[root@nodo20 ~]# grep OPTS /etc/sysconfig/network-scripts/ifcfg-bond0
BONDING_OPTS="mode=802.3ad miimon=100"
```

**Comment by gurevichmark** [comment]
@conxuro Have you tried something like:
`chdef nodo20 nicextraparams.bond0=BONDING_OPTS="lacp_rate=1"`

**Comment by cxhong** [comment]
I can recreate this on our rhels8 test cluster,  will work on this issue

**Comment by conxuro** [comment]
I have just test `chdef nodo20 nicextraparams.bond0=BONDING_OPTS="lacp_rate=1"` (and without ") and only with this option the result is:
```
[root@nodo20 ~]# grep OPTS /etc/sysconfig/network-scripts/ifcfg-bond0
BONDING_OPTS="lacp_rate=1"
```
However if I add more than one parameter it only adds the first one:
```
$ chdef nodo20 nicextraparams.bond0=BONDING_OPTS="lacp_rate=1 mode=802.3ad"
[root@nodo20 ~]# grep OPTS /etc/sysconfig/network-scripts/ifcfg-bond0
BONDING_OPTS="lacp_rate=1"

$ chdef nodo20 nicextraparams.bond0=BONDING_OPTS="mode=802.3ad lacp_rate=1"
[root@nodo20 ~]# grep OPTS /etc/sysconfig/network-scripts/ifcfg-bond0
BONDING_OPTS="mode=802.3ad"
```

**Comment by cxhong** [comment]
right,  have to use different string delimiter,  ` `, `,`, `|` are all not working here.

**Comment by gurevichmark** [comment]
Fixed by #6599
